smart_card_connector_app/src/window.css

```diff
@@ -26,26 +26,22 @@ button {
   font-weight: 400;
 }
 
-#title::after {
-  content: '__MSG_mainWindowTitle__';
-}
-
 body {
   margin: 0;
   padding: 0 0 0 16px;
 }
 
-#header {
+#navbar {
```

Comment:
OK, but please make sure that neither the header nor the close button moved after this change.